lms/djangoapps/teams/tests/test_views.py

```diff
@@ -925,6 +933,14 @@ def test_not_enrolled_in_team_course(self, user):
         )
         self.assertIn('not enrolled', json.loads(response.content)['developer_message'])
 
+    def test_over_max_team_size(self):
+        response = self.post_create_membership(
+            400,
+            self.build_membership_data('student_enrolled_other_course_not_on_team', self.test_team_5),
```

Comment:
I know the test was already written this way, but it would be helpful to have the team name reference which course it is in (test_team_in_course_2 or something like that).